README.md

```diff
@@ -98,9 +98,9 @@ Open the solution in Visual Studio to configure the projects
 
 #### Configure the sample to use your Azure AD app
 
-   1. In the `App.xaml.cs` file from th cloned repo, set your application/client ID copied from the App Registration Portal where you will have registered an application and added the native platform.
+   1. In the `MainPage.xaml.cs` file from th cloned repo, set your application/client ID copied from the App Registration Portal where you will have registered an application and added the native platform.

```

Comment:
@bgavrilMS : that's the kind of changes which for sure will break:
- the portal
- the app creation scripts.
@andretms to confirm